hal/armv7m/stm32/l4/timer.c

```diff
@@ -163,19 +165,45 @@ void timer_setAlarm(time_t us)
 		ticks = (ARR_VAL * 2) / 3;
```

Comment:
This is no longer needed as it implies that (setval > ARR_VAL)

Reply:
Overflow check still needed, sightly reformatted the condition